vrecord

```diff
@@ -809,8 +809,8 @@ _lookup_choice(){
         "24-bit PCM")
             AUDIOCODECNAME="24-bit PCM"
             MIDDLEOPTIONS+=(-c:a pcm_s24le) ;;
-        "FLAC")
-            VIDEOCODECNAME="FLAC"
+        "24-bit FLAC")
+            VIDEOCODECNAME="24-bit FLAC"
```

Comment:
I'm not sure that would happen. Wouldn't the output as written be 32 bit?

Reply:
just tested, and mediainfo reports 24. but i clearly screwed something up on line 813

Reply:
I can 813 to AUDIOCODECNAME (duh), it seems to work, but I'm getting a broken pipe message (file seems good tho):

`
generating QCTools report... done
av_interleaved_write_frame(): Broken pipe
Error writing trailer of pipe:: Broken pipe
frame=  320 fps= 29 q=-0.0 Lq=-0.0 q=-1.0 size=    3168kB time=00:00:10.67 bitrate=2430.7kbits/s speed=0.978x    
video:731654kB audio:16513kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
Conversion failed!
`

Reply:
`VIDEOCODECNAME="24-bit FLAC"`
@bturkus plz fix